bucket/onionshare.json

```diff
@@ -0,0 +1,30 @@
+{
+    "version": "2.1",
+    "description": "Securely and anonymously send and receive files.",
+    "homepage": "https://onionshare.org/",
+    "license": "GPL-3.0-only",
+    "url": "https://github.com/micahflee/onionshare/releases/download/v2.1/onionshare-2.1-setup.exe#/dl.7z",
+    "hash": "sha512:395299aa40d8e15066edfd898d7633d283a9cf3a304bf25a06af62c57b79fe767a48b173957ffbddacbffa27cfcbb71013e917134fb003424abc6995f0602943",
+    "bin": [
+        "onionshare-gui.exe",
```

Comment:
Does gui have some sort of command line suport?

Reply:
No, it's my fault, I forget to add `[]` ...